runbotics-orchestrator/src/main/java/com/runbotics/domain/Process.java

```diff
@@ -53,18 +54,12 @@ public class Process implements Serializable {
     @Column(name = "updated")
     private ZonedDateTime updated;
 
-    @Column(name = "executions_count")
-    private Long executionsCount;
```

Comment:
does the guest 10 runs limit still works?

Reply:
Yup.